ocaml/idl/datamodel_errors.ml

```diff
@@ -277,7 +277,7 @@ let _ =
   error Api_errors.vgpu_type_not_compatible_with_running_type ["pgpu"; "type"; "running_type"]
     ~doc:"VGPU type is not compatible with one or more of the VGPU types currently running on this PGPU" ();
   error Api_errors.vgpu_type_not_compatible ["type"]
-    ~doc:"You tried to create a VGPU that is not compatible with existing types on the VM." ();
+    ~doc:"You tried to create a virtual GPU that is not compatible with the existing types on the VM." ();
```

Comment:
This "You tried" bit should be removed completely (from all the messags in this file really) because it is not user friendly. It should read instead "Cannot create a virtual GPU that is incompatible with the existing types on the VM."